.github/workflows/ci.yml

```diff
@@ -64,6 +69,12 @@ jobs:
       - name: Clone starter
         run: git clone https://github.com/vtex-sites/starter.store.git starter
 
+      - name: Install specific Yarn version
+        run: npm install -g yarn@1.22.19 # https://github.com/yarnpkg/yarn/issues/9015
```

Comment:
What would happen if the developer was using another version of yarn?

Reply:
This is a issue with yarn itself. It only happens on CI (running corepack on subfolders)

Reply:
I saw the issue now. It makes sense now.